wallets/provider-rabby/src/index.ts

```diff
@@ -0,0 +1,69 @@
+import type {
+  CanEagerConnect,
+  CanSwitchNetwork,
+  Connect,
+  Subscribe,
+  SwitchNetwork,
+  WalletInfo,
+} from '@rango-dev/wallets-shared';
+import type { BlockchainMeta, SignerFactory } from 'rango-types';
+
+import {
+  canEagerlyConnectToEvm,
+  canSwitchNetworkToEvm,
+  getEvmAccounts,
+  subscribeToEvm,
+  switchNetworkForEvm,
+  WalletTypes,
+} from '@rango-dev/wallets-shared';
+import { evmBlockchains } from 'rango-types';
+
+import { rabby as rabby_instance } from './helpers';
+import signer from './signer';
+
+export const config = {
+  type: WalletTypes.Rabby,
+};
+
+export const getInstance = rabby_instance;
+export const connect: Connect = async ({ instance }) => {
+  /*
+   * Note: We need to get `chainId` here, because for the first time
+   * after opening the browser, wallet is locked, and don't give us accounts and chainId
+   * on `check` phase, so `network` will be null. For this case we need to get chainId
+   * whenever we are requesting accounts.
+   */
+  const { accounts, chainId } = await getEvmAccounts(instance);
+
+  return {
+    accounts,
+    chainId,
+  };
+};
+
+export const subscribe: Subscribe = subscribeToEvm;
+
+export const switchNetwork: SwitchNetwork = switchNetworkForEvm;
+
+export const canSwitchNetworkTo: CanSwitchNetwork = canSwitchNetworkToEvm;
+
+export const getSigners: (provider: any) => SignerFactory = signer;
+
+export const canEagerConnect: CanEagerConnect = canEagerlyConnectToEvm;
+
+export const getWalletInfo: (allBlockChains: BlockchainMeta[]) => WalletInfo = (
+  allBlockChains
+) => {
+  const evms = evmBlockchains(allBlockChains);
+  return {
+    name: 'Rabby',
+    img: 'https://raw.githubusercontent.com/rango-exchange/assets/main/wallets/rabby/rabby.svg',
```

Comment:
Please change the link here. @RyukTheCoder 
```suggestion
    img: 'https://raw.githubusercontent.com/rango-exchange/assets/main/wallets/rabby/icon.svg',
```

Reply:
Done